src/reformat.c

```diff
@@ -485,8 +513,9 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image, avifRGBIm
     const float kr = state->kr;
     const float kg = state->kg;
     const float kb = state->kb;
-    const float * const unormFloatTableY = state->unormFloatTableY;
-    const float * const unormFloatTableUV = state->unormFloatTableUV;
+    float * unormFloatTableY = NULL;
+    float * unormFloatTableUV = NULL;
+    AVIF_CHECKERR(avifCreateYUVToRGBLookUpTables(&unormFloatTableY, &unormFloatTableUV, image->depth, state), AVIF_RESULT_OUT_OF_MEMORY);
```

Comment:
with some synchronization on the init, this could be done once to a global, though that isn't done elsewhere in the lib currently.

Reply:
For most use cases `avifImageYUVToRGBLibYUV()` avoids the need for lookup tables. I do not think it is worth it to save that work to a global.

On a related note, I plan to do the computation without lookup tables based on some heuristic, such as `sample_count < lut_size`, in another PR.